build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.88.0-alpha1'
+    gutenbergMobileVersion = '5445-79fcbebc7cfafe432d7e658bb721754b452d533e'
```

Comment:
Friendly note: we'll need to update this to the `v1.88.0` tag once it is published.